ui/scripts/network.js

```diff
@@ -1612,6 +1613,36 @@
                                                     });
                                                 }
                                             });
+
+                                            if (!isConfigRulesMsgShown) {
+                                                isConfigRulesMsgShown = true;
+                                                $.ajax({
+                                                    url: createURL('listNetworkOfferings'),
+                                                    data: {
+                                                        id: args.context.networks[0].networkofferingid
+                                                    },
+                                                    dataType: 'json',
+                                                    async: true,
+                                                    success: function(json) {
+                                                        var response = json.listnetworkofferingsresponse.networkoffering ?
+                                                        json.listnetworkofferingsresponse.networkoffering[0] : null;
+
+                                                        if (response != null) {
+                                                            var egressdefaultpolicy;
+
+                                                            if (response.egressdefaultpolicy == true) {
+                                                                egressdefaultpolicy = 'Block';
+                                                            } else {
+                                                                egressdefaultpolicy = 'Allow';
+                                                            }
+
+                                                            cloudStack.dialog.notice({
+                                                                message: _l('Configure the rules to ' + egressdefaultpolicy + ' Traffic.')
```

Comment:
Would be great if you can use a localization key from messages.properties (by adding the sentence inside) instead of hard code message to allow the localization of the web UI. Thanks